textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/keywordmatch/KeywordMatcherTest.java

```diff
@@ -175,15 +126,15 @@ public void testKeywordMatcher() throws Exception {
     @Test
     public void testSingleWordQueryInStringField() throws Exception {
         //Prepare Query
-        String query = "Bruce";
+        String query = "bruce";
```

Comment:
In the first Comment by @chenlica, "bruce" has to be an exact match, because it is on StringField.